sky/shell/platform/ios/semantics_listener_impl.h

```diff
@@ -0,0 +1,101 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SKY_SHELL_PLATFORM_IOS_SEMANTICS_LISTENER_IMPL_H_
+#define SKY_SHELL_PLATFORM_IOS_SEMANTICS_LISTENER_IMPL_H_
+
+#include <map>
+#include <UIKit/UIKit.h>
```

Comment:
I generally don't like to include headers with ObjC in our own headers. This makes it impossible to include `semantics_listener_impl.h` from a `.cc` file. Besides, in this case, you are only importing this umbrella header for `CGRect` which you can do without.